### ISSUE TITLE
Load multi structures?

### ISSUE THREAD

**Issue by jourmore** [pragmatic-streamlit/streamlit-molstar#6]
This is a great plugin! Can streamlit-molstar load multi structures?

**Comment by jourmore** [comment]
This is a great plugin! Can streamlit-molstar load multi structures?

**Comment by mapix** [comment]
The Trajectory and Volume categories can now be freely rendered across multiple files through the new auto module (version >=0.4.1). The usage is as follows:

```
    import streamlit as st
    from streamlit_molstar.auto import st_molstar_auto

    import streamlit as st

    st.set_page_config(layout="wide")

    st.write("from remote url")
    files = ["https://files.rcsb.org/download/3PTB.pdb", "https://files.rcsb.org/download/1LOL.pdb"]
    st_molstar_auto(files, key="6", height="320px")

    st.write("from local file")
    files = ['examples/7bcq.pdb', "examples/7bcq.mrc"]

    st_molstar_auto(files, key="7", height="320px")
```

**Comment by mapix** [comment]
fixed some bugs in version >=0.4.6

**Comment by jourmore** [comment]
> fixed some bugs in version >=0.4.6

Thank you very much !!!
